src/cloud-functions.ts

```diff
@@ -417,7 +417,7 @@ export function makeCloudFunction<EventData>({
         },
       });
       if (!_.isEmpty(labels)) {
-        trigger.labels = labels;
+        trigger.labels = { ...labels, ...trigger.labels };
```

Comment:
I think I'd actually flip this. I'd rather bugs show up in the form of "Why can't I set label X" as opposed to "Why am I seeing arbitrary mysterious behavior that depended on a reserved label"